.github/workflows/ci.yml

```diff
@@ -2,12 +2,25 @@
 
 name: Continuous Integration
 
-on: pull_request
+on: push
```

Comment:
the `pull_request` event injects insufficient git information so we'd have to change to this